README.md

```diff
@@ -1,11 +1,21 @@
 <picture>
-  <source media="(prefers-color-scheme: dark)" srcset="./.github/assets/WEBlsp_logo_dark.svg" />
-  <img alt="WEBlsp logo" src="./.github/assets/WEBlsp_logo_light.svg" />
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/web-lsp/weblsp/refs/heads/main/.github/assets/WEBlsp_logo_dark.svg" />
+  <img alt="WEBlsp logo" src="https://raw.githubusercontent.com/web-lsp/weblsp/refs/heads/main/.github/assets/WEBlsp_logo_light.svg" />
 </picture>
 
 A better Language Server for the Web, made with Rust.
```

Comment:
```suggestion
A Language Server for the Web
```

"better" is too subjective, and as time goes on it'll be less and less relevant to compare ourselves to other projects, so might as well keep it simple right now (it also matches Biome's, which is kinda funny)

I don't think the programming language matters too much here, we're past the novelty factor for Rust that made it cool to use the language in READMEs. Even evangelists are starting to find it more annoying. I think dropping it from README and branding early on will force us to focuses on actual benefits.